devel/openssl3/Portfile

```diff
@@ -74,7 +71,7 @@ if {${os.platform} eq "darwin" && ${os.major} < 18} {
                             sha256  6ae015467dabf0469b139ada93319327be24b98251ffaeceda0221848dc09262 \
                             size    15663524
 
-        patchfiles-replace  e9d7083e241670332e0443da0f0d4ffb52829f08.patch \
+        patchfiles-append   e9d7083e241670332e0443da0f0d4ffb52829f08.patch \
                             7e4d731b1c07201ad9374c1cd9ac5263bdf35bce.patch
```

Comment:
The `patchfiles-append` needs to only list the second patch `7e4d731b1c07201ad9374c1cd9ac5263bdf35bce.patch`. The `e9d7083e241670332e0443da0f0d4ffb52829f08.patch` was for 3.2.1, `7e4d731b1c07201ad9374c1cd9ac5263bdf35bce.patch` is for 3.1.5.